website/docs/tutorial/1c-setting-up-databricks.md

```diff
@@ -5,10 +5,244 @@ id: setting-up-databricks
+
+3. Click on `Create Table`. You should see that you have one database named `default`. This is where we will be loading our data into.
+
+    <Lightbox src="/img/databricks_tutorial/images/create_table_upload.png" title="Create Table in Databricks" />
+    
+4. Now let's upload our first csv. Drag and drop the `jaffle_shop_customers.csv` file into the UI. 
+
+
+<Lightbox src="/img/databricks_tutorial/images/create_new_table.png" title="Upload csv file" />
+
+5. Click on `Create Table with UI` and then select your cluster from the dropdown of available clusters. Click `Preview Table`.
+
+    <Lightbox src="/img/databricks_tutorial/images/select_cluster_to_preview_table.png" title="Select Cluster to Preview Table" />
+
+
+6. Update the Table Attributes:
+- Change the table name to `jaffle_shop_customers`
+- Check off the `First row is header`. 
+- Check off `Infer schema`
+- Click on `Create Table` when you are done.
+
+7. Congrats you have successfully uploaded your first table! 
+
+    <Lightbox src="/img/databricks_tutorial/images/new_table_created.png" title="Jaffle Shop Customers table" />
+    
+8. Now let’s do the same for `Jaffle Shop Orders` and `Stripe Payments` by clicking on the Data tab on the left hand console. The end result will be three tables in the default database. While updating the table attributes, follow the same directions as in Step 6 with the following renaming:
+
+- Stripe Payments = `stripe_payments` 
+
+- Jaffle Shop Orders = `jaffle_shop_orders`
+
+    <Lightbox src="/img/databricks_tutorial/images/raw_tables_created.png" title="Raw Data Tables" />
+
 ## Enable a SQL endpoint for Databricks
 
+For dbt to connect to the most optimized SQL experience in Databricks, we need to create an SQL Endpoint for our connection.
+
+1. Up to this point, we have been working in the Data Science & Engineering space. Now we need to change over to SQL.  Find the drop down menu and toggle into the SQL space.
+
+<Lightbox src="/img/databricks_tutorial/images/go_to_sql.png" title="SQL space" />
+
+2. We will be setting up a SQL endpoint now.  Select `SQL Endpoints` from the left hand side console.  You will see that a default SQL Endpoint exists.  
+
+<Lightbox src="/img/databricks_tutorial/images/sql_endpoints.png" title="SQL Endpoints" />
+
+3. Click `Start` on the Starter Endpoint.  This will take a few minutes to get the necessary resources spun up.
+
+4. Once that's done, let’s ensure that we can query the training data that we loaded earlier.  Navigate back to the SQL space through the left hand menu and choose `SQL Editor`.  This will bring you to a query editor.
+
+5. Ensure that you can run a select * from each of the tables with the following code snippets. 
+
+```sql
+select * from default.jaffle_shop_customers
+```
+
+```sql
+select * from default.jaffle_shop_orders
+```
+
+```sql
+select * from stripe.payments
+```
+
```

Comment:
suggestion: I don't know how the formatting will change if we removed the whitespace between these.  I'm torn here.  We can make them separate snippets to make them distinct or rely on one code block and assume the learner knows that they are separate - what you do you think?

Reply:
I can remove the whitespaces; it was just easier for me to write code wish. We want to keep them as separate snippets because I couldn't figure out how to get the Databricks UI to run just one query if 3 are in the window. So I want to really emphasize that these should be run separately (and not use the `;` as a hint).

Reply:
that works! so overall one code block, with 1 line of space between the each `select *`.  Sounds good!